src/views/Settings.vue

```diff
@@ -2,42 +2,96 @@
   <ion-page>
     <ion-header :translucent="true">
       <ion-toolbar>
-        <ion-title>{{ $t("Settings") }}</ion-title>
+        <ion-title>Store Settings</ion-title>
       </ion-toolbar>
     </ion-header>
     
     <ion-content :fullscreen="true">
-      <!-- Profile of user logged in -->
       <ion-item>
-        <ion-icon :icon="personCircleOutline" slot="start" />
-        <ion-label>{{ userProfile !== null ? userProfile.partyName : '' }}</ion-label>
-        <ion-button slot="end" fill="outline" color="dark" @click="logout()">{{ $t("Logout") }}</ion-button>
+        <ion-label>Store</ion-label>
+        <ion-select placeholder="store name">
+          <ion-select-option>Broadway</ion-select-option>
+          <ion-select-option>Garden City</ion-select-option>
+        </ion-select>
       </ion-item>
 
-      <!-- Select store -->
-       <ion-item>
-        <ion-icon :icon="storefrontOutline" slot="start" />
-        <ion-label>{{ currentFacility.facilityId ? currentFacility.facilityId : '' }}</ion-label>
-      <ion-select interface="popover" :placeholder="$t('store name')" :selected-text="currentFacility.facilityId" @ionChange="setFacility($event)">
-        <ion-select-option v-for="facility in userProfile.facilities" :key="facility.facilityId" :value="facility.facilityId" >{{ facility.facilityId }}</ion-select-option>
-      </ion-select>
-      </ion-item>
+      <ion-card>
+        <ion-item lines="none">
+          <ion-label class="text-wrap">Documents to print when packing orders</ion-label>
+        </ion-item>
+        <ion-item>
+          <ion-label>Shipping label</ion-label>
+          <ion-checkbox slot="end" />
+        </ion-item>
+        <ion-item lines="none">
+          <ion-label>Packing slip</ion-label>
+          <ion-checkbox slot="end" />
+        </ion-item>
+      </ion-card>
+
+      <ion-card>
+        <ion-item lines="none">
+          <ion-label>Fulfillment: On</ion-label>
+        </ion-item>
+        <ion-item lines="none">
+          <ion-label class="text-wrap">Broadway has 47 outstanding orders and 77 in progress orders.</ion-label>
+        </ion-item>
+        <div class="actions">
+          <div class="desktop-only">
```

Comment:
move desktop only class to actions div instead of repeating it on children

Reply:
Done